controllers/user.js

```diff
@@ -229,7 +229,9 @@ async function getDashboard(req, res) {
   const user = req.user;
   const allBreaches = req.app.locals.breaches;
   const { verifiedEmails, unverifiedEmails } = await getAllEmailsAndBreaches(user, allBreaches);
+  const utmOverrides = getUTMContents(req);
```

Comment:
Setting the `/share/` link catcher on this page too, in case the person who uses it is already signed into Monitor.